### PR TITLE
[Translation zh-tw] Fundamentals > Related Guides

### PR DESCRIPTION
**Pull request by kennith**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/434)

### CODE REVIEW

File: src/content/zh-tw/_shared/related-guides/service-workers.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
```
Comment (petele) [inline src/content/zh-tw/_shared/related-guides/service-workers.html]: Do not add doctype or other extra tags

**Comment by petele** [comment]
@agektmr, @kennith, @limhenry - it looks like GitLocalize is adding extra stuff to the files, in this case it's added a doctype, html and body elements, etc.

**Comment by agektmr** [comment]
@sotayamashita @chikathreesix can you take a look?

**Comment by sotayamashita** [comment]
@agektmr I'll check it out.

**Comment by sotayamashita** [comment]
@kennith I would appreciate if you could remove extra code yourself because I cannot find branch on your forked repo.

**Comment by petele** [comment]
Any updates on this?

**Comment by kennith** [comment]
@sotayamashita I cannot reproduce the repo. Can I resubmit a pull request from GitLocalize?

**Comment by petele** [comment]
Closing due to inactivity.

**Pull request closed without merging**